site/en/blog/permissions-chip/index.md

```diff
@@ -0,0 +1,145 @@
+[notification permission data](https://developers.google.com/web/updates/2020/02/notification-permission-data-in-crux)
+in the Chrome UX report yourself. For now, consider the table below that shows
+how Windows users reacted to the geolocation and the notification prompts on sites in an accumulated way:
+
+<div class="table-wrapper scrollbar">
+<table>
+<thead>
+<tr>
+<th>Action</th>
+<th>Percent of geolocation prompts</th>
+<th>Percent of notification prompts</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>Allow</td>
+<td>8.09%</td>
+<td>6.69%</td>
+</tr>
+<tr>
+<td>Block</td>
+<td>6.47%</td>
+<td>9.20%</td>
+</tr>
+<tr>
+<td>Dismiss</td>
+<td>36.12%</td>
+<td>35.76%</td>
+</tr>
+<tr>
+<td>Ignore</td>
+<td>49.22%</td>
+<td>47.19%</td>
+</tr>
+</tbody>
+</table>
+</div>
+
+Given an ignore/dismiss rate of approximately 85%, and especially given how much the prompt stands out and
+insists on users making a decision immediately, there is a conflict between the level of urgency
+assumed by the browser versus the user's preference for waiting to make a decision. This creates the
+perception that it's "annoying" for a site to ask for a permission.
+
+## New design
+
+From Chrome 97 on, we therefore introduced an animated chip UI which appears next to the lock
+whenever a permission is requested. This consists of an icon and label describing the permission being
+requested. Our aim was to improve the experience of web browsing while avoiding permission requests
+that are generally unnecessary for the vast majority of users and frequently ignored or dismissed.
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/SsFtKKnZzbkAovNkX4Lt.png", alt="Google search engine results page showing a search for pizza and the permissions chip with 'Use your location?' label.", width="653", height="229" %}
+
+The existing prompt bubble will be shown when the request chip is clicked (if not already shown) and
+the request UI is automatically augmented with the request bubble based on the heuristics listed
+below:
+
+- The permission was triggered through a user gesture when interacting with the site itself rather
+  than automatically triggered by the site.
+- The permission is deemed essential and generally non-spammy. Currently this includes camera,
+  microphone, and camera paired with microphone.
+
+### Flow of the new design
+
+#### Without user gesture
+
+For non-essential permissions not triggered by a gesture, the prompt no longer intrudes on the site
+contents, and does not insist on an immediate decision. The user can ignore the request chip until
+they have enough information to make a decision.
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/sLmK0PKtJzSLegl7VBLM.png", alt="Flow diagram going from padlock to the geolocation prompt, which, if dismissed, results in the 'geolocation blocked' icon, which, after a four second delay, is finally replaced with the padlock again.", width="800", height="292" %}
```

Comment:
Maybe these flow images can be without the header text. I can grab these from slide deck. 

Also currently we don't transition from the lock to the chip, although that is the future intention.

Reply:
If you can send me the SVGs, this would be great.